src/components/Banner/__snapshots__/Banner.test.tsx.snap

```diff
@@ -34,7 +34,7 @@ exports[`<Banner /> Brand story renders snapshot 1`] = `
         Summit Learning has a full-time team dedicated to constantly improving our curriculum. To see the updates,
          
         <button
-          class="clickable-style button button--link clickable-style--link clickable-style--neutral"
+          class="clickable-style clickable-style--link clickable-style--neutral button button--link"
```

Comment:
These snapshot changes are from moving the `className` prop to the bottom of the class list.